interface-definitions/system_option.xml.in

```diff
@@ -49,6 +49,26 @@
                   <valueless/>
                 </properties>
               </leafNode>
+              <leafNode name="amd-pstate-driver">
+                <properties>
+                  <help>Enables and configures pstate driver for AMD Ryzen and Epyc CPUs</help>
+                  <completionHelp>
+                    <list>active passive guided</list>
+                  </completionHelp>
+                  <valueHelp>
+                    <format>active</format>
+                    <description>This is the low-level firmware control mode based on the profile set and the system governor has no effect</description>
```

Comment:
I think these descriptions are too wordy.
```suggestion
                    <description>Give all control to the pstate driver instead of the system governor</description>
```

Reply:
This is not quite correct as it's not the p-state driver which is in control. I'll try to reword it.